src/main/java/de/tum/in/www1/artemis/web/rest/LectureResource.java

```diff
@@ -117,7 +121,7 @@ public ResponseEntity<Lecture> createLecture(@RequestBody Lecture lecture) throw
 
         Lecture savedLecture = lectureRepository.save(lecture);
         channelService.createLectureChannel(savedLecture, Optional.ofNullable(lecture.getChannelName()));
-
+        helpExecuteIngestionPipeline(savedLecture);
```

Comment:
Is this needed here? Afaik you cannot upload a file while creating the lecture, so there should not be anything to ingest, right?

Reply:
If you import a lecture that has already lecture units with files, they should be ingested as well right ?

Reply:
But this is when you create a lecture. Importing is a different endpoint.